spec/features/merchants/index_spec.rb

```diff
@@ -66,5 +67,24 @@
 
       expect(page).to have_content("Merchant has been disabled")
     end
+
+    it "merchant items deactivated once disabled" do
+      visit "/login"
+
+      fill_in :email, with: @user_1.email
+      fill_in :password, with: @user_1.password
+      click_button "Log In"
+
+      visit "/admin/merchants"
+
+      within "#merchant-#{@bike_shop.id}" do
+        click_button "Disable"
+        expect(current_path).to eq("/admin/merchants")
+        expect(page).to have_content("Disabled: true")
+      end
+
+      expect(@bike_shop.items.first.active?).to eq(false)
+      expect(@bike_shop.items.last.active?).to eq(false)
```

Comment:
Test that when a merchant is disabled, the merchant's items are no longer active